src/templates/fr/2019/accessibility_statement.html

```diff
@@ -0,0 +1,123 @@
+        </p>
+
+        <p>
+          Nous nous considérons partiellement conformes parce que nous n’avons pas effectué un audit complet et vérifié de l’extérieur du site Web, ni que le site Web n’a été examiné par les utilisateurs de lecteurs d’écran, nous ne pouvons donc pas être sûrs de notre état actuel. Nous avons audité toutes les pages du site Web à l’aide d’un certain nombre d’outils, notamment <a href="https://wave.webaim.org/" hreflang="en">l’outil d’accessibilité WAVE</a>, <a href="https://hacks.mozilla.org/2019/10/auditing-for-accessibility-problems-with-firefox-developer-tools/" hreflang="en">l’inspecteur d’accessibilité Firefox</a> et le <a href="https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd" rel="nofollow">plugin AX Chrome</a>.
+        </p>
+
+        <p>
+          Le but de l’Almanach Web est de fournir des informations détaillées sur l’état du Web. Bien que nous cherchions à nous assurer que tout le contenu est facilement lisible, même par des non-experts, la nature du matériel peut signifier qu’il peut y avoir un niveau d’expertise et de compréhension technologique nécessaire pour comprendre certains chapitres et certaines mesures, qui peuvent ne pas répondre au <a href="https://www.w3.org/WAI/WCAG21/quickref/#reading-level" hreflang="en">niveau WCAG 2.1. des directives de niveau de lecture AAA</a>.
+        </p>
+
+        <p>
+          Nous utilisons également Google Sheets pour certaines visualisations interactives, qui peuvent également ne pas être conformes aux normes d’accessibilité WCAG. Ceux-ci sont chargés via des <code>iframes</code>, nous fournissons des images pour toutes nos figures et affichons uniquement des visualisations interactives lorsque le logiciel client le prend en charge. Si vous utilisez un navigateur Web et que vous ne souhaitez pas afficher les visualisations interactives, les images peuvent être appliquées en ajoutant un paramètre <code>?print</code> pour tous les chapitres, <a href="./fr/2019/accessibility?print">consultez le chapitre Accessibilité en version imprimable</a> pour un exemple de ce type. Nous pouvons chercher à rendre cette fonctionnalité plus facile à activer si nécessaire, mais toutes les visualisations ont des descriptions textuelles détaillées, nous ne pensons donc pas que cela soit nécessaire. <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">Veuillez nous faire savoir</a> si nous nous trompons à ce sujet.
```

Comment:
Visualisations => celles-ci
+ reformulation dernière phrase

```suggestion
          Nous utilisons également Google Sheets pour certaines visualisations interactives, qui peuvent également ne pas être conformes aux normes d’accessibilité WCAG. Celles-ci sont chargées via des <code>iframes</code>, nous fournissons des images pour toutes nos illustrations et affichons uniquement des visualisations interactives lorsque le logiciel client le prend en charge. Si vous utilisez un navigateur Web et que vous ne souhaitez pas afficher les visualisations interactives, les images peuvent être appliquées en ajoutant un paramètre <code>?print</code> pour tous les chapitres, <a href="./fr/2019/accessibility?print">consultez le chapitre Accessibilité en version imprimable</a> pour un exemple de ce type. Nous pouvons chercher à rendre cette fonctionnalité plus facile à activer si nécessaire, mais toutes les visualisations ont des descriptions textuelles détaillées, nous ne pensons donc pas que cela soit nécessaire. Si vous pensez que c'est une erreur, <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">n'hésitez pas à nous le dire</a>.
```